app/generated_resources.grd

```diff
@@ -1713,15 +1713,15 @@ are declared in tools/grit/grit_rule.gni.
       </if>
       <if expr="not is_macosx">
         <message name="IDS_ABANDON_DOWNLOAD_DIALOG_INCOGNITO_MESSAGE" desc="Message on a dialog shown when the user closes the last incognito window while one or more downloads are in progress. This string is shown on Windows, Brave OS, and Linux, which all use 'Exit' to refer to closing a browser.">
-          Exit incognito mode anyway?
+          Exit private window mode anyway?
```

Comment:
is the casing on this one correct? or should it be `Private Window` (similar to the quit message below)

Reply:
(I guess since it's not macOS, it wouldn't be in title case, so it's probably OK)